launch/sim2play.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script is extremely similar to real2play.bash. See that script for details.
+# this exhibits weird behavior on CTRL-C / close because the method in real2play.bash didn't work quite as intended
```

Comment:
I noticed that the stack is not really responding to the KeyboardInterrupt like in make run-sim. It's rather terminating after waiting for a 5 second threshold. This is more apparent when you run sim2play without the ER-Force simulator.
![Screenshot from 2022-10-02 19-59-58](https://user-images.githubusercontent.com/44911049/193482423-3a534f65-0e8b-4263-a608-400e9305f229.png).

Reply:
As discussed IRL, this is not ideal but not a breaking change either, and the changes I made to soccer.launch.py broke the old sim2play.launch.py.